microcosm_daemon/health_reporter.py

```diff
@@ -35,7 +35,11 @@ def __call__(self, health, prev_health, errors):
             if isinstance(error, ExitError):
                 continue
 
-            logger.warn(f"Caught error during state evaluation: {error}", exc_info=True)
+            logger.warning(
+                "Caught error during state evaluation",
+                extra=dict(error=error),
+                exc_info=True,
```

Comment:
The linter is complaining about the logging format based on our own rule that log messages shouldn't use f-strings (see [CircleCI](https://app.circleci.com/pipelines/github/globality-corp/microcosm-daemon/89/workflows/178dd05f-43df-409c-8cf9-3807d3705935/jobs/600)). I'm not sure why that didn't show up in the original PR.